dice/platform_adapter_gocq.go

```diff
@@ -273,6 +273,13 @@ func FormatDiceIDQQChGroup(guildID, channelID string) string {
 	return fmt.Sprintf("QQ-CH-Group:%s-%s", guildID, channelID)
 }
 
+func isLink(text string) bool {
+	// 正则表达式匹配三种情况：file URI、http(s) URL 和 base64 URI
+	regex := `^(file:\/\/\/[^\s]+|(http|https):\/\/[^\s]+|base64:\/\/[a-zA-Z0-9+/=]+)$`
```

Comment:
简单些，直接判断是否存在 URL 的 scheme 部分

```suggestion
	regex := `^[a-z]+://`
```

Reply:
该判断是根据GO-CQHTTP对file可填写的描述来的，确认这样没有问题的话我没有意见？

Reply:
显然 gocq 对 cq 码的规定是不可靠的, gocq 本身都没有遵守自己的规定